### DISCUSSION TITLE
Is there a way to create a volume where `fromBackup` points to the latest backup, without knowing which one?

### DISCUSSION THREAD

**Discussion by marvelous** [longhorn/longhorn#3811] [Q&A]
I'd like to provision a new test cluster with the latest backups. Currently I have to specify a particular backup with something like `fromBackup: 'nfs://mybackups:/mybackups?volume=myvolume&backup=xyz'`, which requires me to lookup which backup is the latest and write that to the git repo that ArgoCD will sync from.

Is there a way for Longhorn to go look into the backup directory (it's a new cluster, so Longhorn doesn't have backup metadata at hand), determine which backup is the latest and use that?

**Comment by PhanLe1010** [comment]
cc @derekbit
